.buildkite/package-lock.json

```diff
@@ -17,9 +17,9 @@
       },
       "devDependencies": {
         "@types/chai": "^4.2.10",
+        "@types/jest": "^29.2.0",
```

Comment:
was this intentional for .buildkite?  the test runner still appears to be mocha

Reply:
Oh, I checked the test files themselves and they appeared to be jest tests 🙃

I wanted to eliminate all @types/mocha instances but maybe I can just leave these ones be.

Reply:
Resolved with [`addee95` (#147284)](https://github.com/elastic/kibana/pull/147284/commits/addee9559956a48c8f2e610d45bfbedd9aa5bba1)